holidays/groups/hindu.py

```
@@ -27,6 +28,13 @@ def __init__(self, cls=None, *, show_estimated=False) -> None:
         self._hindu_calendar = cls() if cls else _HinduLunisolar()
         self._hindu_calendar_show_estimated = show_estimated
 
+    @property
+    def _ram_navami(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_ram_navami`

(ANN202)